pkgs/build-support/setup-hooks/fix-darwin-dylib-names.sh

```diff
@@ -36,5 +36,5 @@ fixDarwinDylibNames() {
 
 fixDarwinDylibNamesIn() {
     local dir="$1"
-    fixDarwinDylibNames $(find "$dir" -name "*.dylib")
+    fixDarwinDylibNames $(find "$dir" -name "*.dylib" -o -name "*.so")
```

Comment:
```suggestion
    fixDarwinDylibNames $(find "$dir" -name "*.dylib" -o -name "*.so" -o -name "*.so.*")
```

* can't be added to `*.so` because it would match .socket

Reply:
I think we still want to match a bare `.so` though?

```suggestion
    fixDarwinDylibNames $(find "$dir" -name "*.dylib" -o -name "*.so" -o -name "*.so.*")
```

Reply:
There's no diff in your suggestion compared to mine

Reply:
You're right, I'm blind as a mole. Apologies.